docs/how-to/contribute-docs.rst

```diff
@@ -83,3 +83,33 @@ setting up a virtual environment and building the documentation.
    #wokeignore:rule=master
 .. _literalinclude: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude
 .. _YAML: https://yaml.org/
+.. _`submit an issue`: https://github.com/canonical/rockcraft/issues/new/choose
+.. _`create a fork`: https://github.com/canonical/rockcraft/fork
+.. _`CONTRIBUTING`: https://github.com/canonical/rockcraft/blob/main/CONTRIBUTING.rst
+
+
+Making a contribution
+---------------------
+
+Report issues
+~~~~~~~~~~~~~
+
+If you have a request or found a problem with the documentation, then please
+`submit an issue`_. These issues are supervised and regularly triaged by the repository
+owners. If you don't receive an answer within 2 weeks, please reply to your own issue
+asking for an update.
+
+Propose changes
+~~~~~~~~~~~~~~~
+
+Community contributions are more than welcome. To become an external contributor you
+should:
+
+1. `create a fork`_ of the Rockcraft repository,
+2. commit the changes to your fork (ideally to a new branch),
+    1. make sure to follow the project's `CONTRIBUTING`_ guidelines,
+3. create a Pull Request against the ``main`` branch.
+
+Similarly to new issues, new Pull Requests (PR) are also supervised and regularly
```

Comment:
"Similarly" here is an adjective not an adverb, so either "similar" or "like" would be more technically correct.

It's not a big deal (and I had to check with a friend that I wasn't mistaken), so if you prefer the way it sounds currently that's fine, since good flow is more important than technical-correctness.